website/docs/index.html.markdown

```diff
@@ -174,7 +174,7 @@ In addition to [generic `provider` arguments](https://www.terraform.io/docs/conf
 * `shared_credentials_file` = (Optional) This is the path to the shared credentials file.
   If this is not set and a profile is specified, `~/.aws/credentials` will be used.
 
-* `token` - (Optional) Use this to set an MFA token. It can also be sourced
+* `token` - (Optional) Temporary session token retrieved from an MFA device. It can also be sourced
```

Comment:
This was certainly confusing before! Thanks for this update. While we are fixing this, it might be preferable to note that session tokens are generally used with _any_ temporary credentials. References:
* https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa.html
* https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html

```md
* `token` - (Optional) Session token for validating temporary credentials. Typically used with identity federation or Multi-Factor Authentication. It can also be sourced from the `AWS_SESSION_TOKEN` environment variable.
```

Reply:
Thanks @bflad!  Let me know if the edits I just made are better or need more clarification...